pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go

```diff
@@ -40,8 +54,9 @@ type NMState struct{}
 
 // +k8s:openapi-gen=true
 type KubeMacPool struct {
-	RangeStart string `json:"rangeStart,omitempty"`
-	RangeEnd   string `json:"rangeEnd,omitempty"`
+	RangeStart string     `json:"rangeStart,omitempty"`
+	RangeEnd   string     `json:"rangeEnd,omitempty"`
+	Placement  *Placement `json:"placement,omitempty"`
```

Comment:
why does KMP need a specific Placement configured? should it be the same as the one set in `HyperConvergedConfig`?

Reply:
At first, I that was my plan to have global hyperconvergedconfig used by all components, but then I realized we have this [default antiPodAffinity in kubemacpool.yaml ](https://github.com/kubevirt/cluster-network-addons-operator/blob/fdc32d0a004b3c44471a0886ee02d1d9fd04207b/data/kubemacpool/kubemacpool.yaml#L252
)

Reply:
is this Anti Affinity constradicting the Placement you're trying to add?
if it is - maybe it is worth to remove it from KMP altogether? @phoracek what do you think?

Reply:
@RamLavi It's not neccessarily contradicting, but if we have one placementconfiguration for all of the components, then by adding this affinity specific for KMP it's also added to all other infra components. I'm not sure if it's desirable.
I agree it would be much easier to only have the placement configured in CNAO.